### PR TITLE
Add guidelines on finishing the day

### LINKED ISSUE #225: Create guidelines for ending the first day
- Do the hack cheer
- Say anything for the following day

### PR DESCRIPTION
**Pull request by zachlatta**
This PR closes #225.

@MaxWofford can I pretty please get your eyes on this (and a merge, if appropriate)?

/cc @jonleung

### CODE REVIEW

File: cohort_4/playbook/activities.md [hunk 1]
```diff
@@ -153,3 +154,16 @@ https://docs.google.com/spreadsheets/d/1D8y_UB11C4QrXB9kn86-UqVQQqT9FUDqftx1AMHI
 |   2 | https://docs.google.com/a/zachlatta.com/forms/d/1_9K3b4Nwgn3-mFqMrrUObwiMHt_53y9LVNyCQAULn7I |
 |   3 | https://docs.google.com/a/zachlatta.com/forms/d/1kgCrsPuTFJ-404JdpFo3acomXmwvTIOJHXzOp_NoUeI |
 |   4 | https://docs.google.com/a/zachlatta.com/forms/d/1ASfJdmXGUVRdGJR6rmzedDQ2EDEL7Hc11JD0xjBJMp8 |
+
+
```
Comment (maxwofford) [inline cohort_4/playbook/activities.md]: Should there be 2 newlines here?

**Comment by maxwofford** [comment]
@zachlatta Made a comment. Other than that it's ready to merge.

**Comment by jonleung** [comment]
Looks good!

**Comment by zachlatta** [comment]
@MaxWofford fixed!

**Pull request merged**